src/ai/backend/manager/api/auth.py

```diff
@@ -428,6 +428,16 @@ async def auth_middleware(request: web.Request, handler) -> web.StreamResponse:
     Fetches user information and sets up keypair, user, and is_authorized
     attributes.
     """
+    allow_list = request.app["auth_middleware_allowlist"]
```

Comment:
I see a lot of other code accessing the `RootContext` field stored in `request.app[“_root.context”]`, is there a reason why you stored it directly in the app here?
I'd like to know what separates the cases that access app directly from those that access RootContext. @achimnol

Reply:
`RootContext` contains mutable configurations such as `local_config` and `shared_config`, as well as many objects that are frequently accessed throughout the codebase.

So I thought it might be better to inject hardcoded values into the `app` instead. 
(But this is just my personal opinion.)